src/ogt_vox.h

```diff
@@ -305,6 +308,9 @@
     // writes the scene to a new buffer and returns the buffer size. free the buffer with ogt_vox_free
     uint8_t* ogt_vox_write_scene(const ogt_vox_scene* scene, uint32_t* buffer_size);
 
+    // calculate the final position for the 3 input coordinates
+    void ogt_vox_apply_transform(const ogt_vox_transform *transform, float pivot_x, float pivot_y, float pivot_z, int i, int j, int k, int *out_i, int *out_j, int *out_k);
```

Comment:
This function doesn't feel like the right level of abstraction for this library - I'd like to remove it.